internal/jobservice/server/server.go

```diff
@@ -43,3 +44,15 @@ func (s *JobServiceServer) GetJobStatus(ctx context.Context, opts *js.JobService
 
 	return response, err
 }
+
+func (s *JobServiceServer) Health(ctx context.Context, _ *types.Empty) (*js.HealthCheckResponse, error) {
+	log.Info("JobService Health Check Starting")
```

Comment:
I think this might create a lot of log noise if there are frequent health checks -- maybe shouldn't log at all

Reply:
This might get noisy, but if it gets on people's nerves, we can quickly dial it down to a `log.Debug()`.

Reply:
Removed.